src/patternfly/components/Alert/alert.hbs

```diff
@@ -1,3 +1,9 @@
 <div class="pf-c-alert {{ pf-c-alert--Modifier }}" aria-label="{{ pf-c-alert--AriaLabel }}">
```

Comment:
lowercase m for modifier, also add a conditional around the modifier. Add the aria-label to the alert-attributes.

Reply:
done